packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/types/ground_overlay.dart

```diff
@@ -282,6 +288,7 @@ class GroundOverlay implements MapsObject<GroundOverlay> {
     bool? visibleParam,
     bool? clickableParam,
     VoidCallback? onTapParam,
+    double? zoomLevelParam,
```

Comment:
Can zoom level be changed after initialization.
I would remove zoomLevelParam and handle it like position,bounds,anchorwidth and height; and only copy the exiting value and do not allow to change it after initialization.